pkg/metrics/real_time_test.go

```diff
@@ -217,10 +223,162 @@ func TestFilterOutContainerNotInPod(t *testing.T) {
 	testDescription := "Successfully filters out containerStats for containers not in pod.spec.containers"
 	t.Run(testDescription, func(t *testing.T) {
 		filterOutContainerNotInPod(podStats, pod)
-		assert.EqualValues(t, len(podStats.Containers), 3)
-		assert.EqualValues(t, podStats.Containers[0].Name, fake_container1)
-		assert.EqualValues(t, podStats.Containers[1].Name, fake_container2)
-		assert.EqualValues(t, podStats.Containers[2].Name, fake_container4)
+		assert.DeepEqual(t, 3, len(podStats.Containers))
+		assert.DeepEqual(t, fake_container1, podStats.Containers[0].Name)
+		assert.DeepEqual(t, fake_container2, podStats.Containers[1].Name)
+		assert.DeepEqual(t, fake_container4, podStats.Containers[2].Name)
 	})
 
 }
+
+func TestGetRealTimeExtensionPodStats(t *testing.T) {
```

Comment:
Would you please run `go fmt`?

Reply:
go fmt did not show any changes. Assuming you meant the empty line in 230, I removed it.